Openshift/documentation/scripts/docDataExtracter.sh

```diff
@@ -90,6 +92,28 @@ oc get identities
 oc get pv > pv.log
 oc get sc > sc.log
 
+###### Gluster related info 
+
+oc get storageclass > storageclass.log
+
+#gluster storage data
+oc get pvc -n default > pvc.log
```

Comment:
Are you trying to get pvc for registry? If yes then add a line in comment. Sorry for being painful.

Reply:
yup will add comment